plugins/providers/docker/driver.rb

```diff
@@ -236,8 +236,20 @@ def all_containers
         execute('docker', 'ps', '-a', '-q', '--no-trunc').to_s.split
       end
 
+      # Attempts to first use the docker-cli tool to inspect the default bridge subnet
+      # Falls back to using /sbin/ip if that fails
+      #
       # @return [String] IP address of the docker bridge
       def docker_bridge_ip
+        bridge = inspect_network("bridge")[0]
+        bridge["IPAM"]["Config"][0]["Gateway"]
```

Comment:
Doing some result type checks here over running into an exception would be preferred: 

```ruby 
bridge = inspect_network("bridge")&.first
if bridge
  bridge_ip = bridge.dig("IPAM", "Config", 0, "Gateway")
end
return bridge_ip if bridge_ip
docker_bridge_ip_fallback
```